src/test/java/qna/domain/QuestionTest.java

```diff
@@ -1,6 +1,32 @@
 package qna.domain;
 
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
 public class QuestionTest {
     public static final Question Q1 = new Question("title1", "contents1").writeBy(UserTest.JAVAJIGI);
     public static final Question Q2 = new Question("title2", "contents2").writeBy(UserTest.SANJIGI);
+
+    @Autowired
+    private QuestionRepository questionRepository;
+
+    @Test
+    void save() {
+        Question actual = questionRepository.save(Q1);
+
+        assertThat(actual).isNotNull();
+    }
+
+    @Test
+    void findById() {
+        Question expected = questionRepository.save(Q2);
+        Question actual = questionRepository.findById(expected.getId()).get();
+
+        assertThat(expected).isEqualTo(actual);
```

Comment:
```suggestion
        Optional<Question> actual = questionRepository.findById(expected.getId());

        assertThat(actual).hasValue(expected);
```

와 같은 방식을 사용하면 Optional도 의미있게 테스트로 활용할 수 있게 됩니다. 참고 부탁드려요!